app/baseComponents/SideMenu/SideMenu.js

```diff
@@ -19,11 +19,13 @@ export type SideMenuEntry = {
   label: string,
   /* eslint-disable-next-line flowtype/no-weak-types */
   iconSrc: any,
+  path: ?string,
   disabled?: boolean
 };
 
 type SideMenuProps = {
   openOnInit?: boolean,
+  onInitSelectedId?: number,
```

Comment:
naming issue:
instead "onInitSelectedId" give this param meaningful name like "initialSelectedId"
also param above "openOnInit" - let's go with convention that boolean params should start with "is/shall" for single decision and "should" for multiple. In this case a proper name would be "isOpenedOnInit"